app/views/hyrax/uploads/_js_templates.html.erb

```diff
@@ -169,6 +187,8 @@
             </button>
         </td>
     </tr>
+  {% var RecentFileId = file.id; %}
+  {% setTimeout(function(){ setVisiblityOptions(RecentFileId) }, 300); %}
```

Comment:
Adding timeout to render the Html via javascript.

Due to the timeout, the `file.id` value was getting a null value. Hence I ended up saving it to a variable and calling the function using the value of the var.